backend/src/main/resources/migration/releases_ddl/mysql/mysql_release_2022_08.up.sql

```diff
@@ -0,0 +1,1338 @@
+-- MySQL dump 10.13  Distrib 8.0.12, for Win64 (x86_64)
```

Comment:
this file will need to be named by the naming scheme that is being used by the migrator. I think it should be
`1_release_2022_08.up.sql` , so that it can be properly ordered by the migration tool to be the first one run.

Reply:
Fixed it